kademlia/storage.py

```diff
@@ -7,6 +7,10 @@
 class IStorage:
     """
     Local storage for this node.
+    IStorage implementations must store and return values as a key value pair.
+    IStorage special behavior (ie: appending to a list instead of setting) must
+    work in only special cases as the default case (key,value pair) is required
+    for the DHT's functioning.
```

Comment:
Definitely appreciate any additional docs!  I'm just not sure what this means.  What is "special behavior" in this context?  What list would be appended to?  DHT's store key/value pairs - that's true - but I'm not sure what these additional lines mean.

Reply:
I tried to make the storage module append items to a list instead of just store values. So for example. set("a",3) , set("a",4), and then get("a") would return [3,4].  However, the IStorage must function only as a key value store because the DHT stores some metadata in itself that it does not want returned as a list. I will try to make it more clear.

Reply:
Yeah - the `set` method mirrors the `STORE` function described in section 2.3 of the paper:

> STORE instructs a node to store a (key, value) pair for later retrieval.